### ISSUE TITLE
tags for grouping similar dats

### ISSUE THREAD

**Issue by ryanramage** [beakerbrowser/hashbase#53]
It would be nice if authors could add one or more tags to their dat meta information, at the same time they set the name and description of the archive. Similar to npm.

And then allow a way to list all the archives of the same tag.

This would allow communities of dats to flourish in finding and publishing to hashbase.

**Comment by pfrazee** [comment]
Yeah that's a good idea. We're going to update hashbase's entire system after we've done more work on Beaker. We want to see how much of hashbase can work entirely by reading the users' dats, rather than using its own private DB.
